org.scala-ide.sdt.core.tests/src/org/scalaide/core/pc/DeclarationPrinterTest.scala

```diff
@@ -142,5 +142,68 @@ class DeclarationPrinterTest {
         exp = exp.tail
       }
     }
+    Assert.assertTrue(exp.isEmpty)
+  }
+
+  @Test
+  def testVar():Unit={
+    runTest("""
+          |var a:Int=4
+          |def f():Unit = {
+          |  println(a/**/)
+          |}
+          |""".stripMargin,
+          "var a: Int")
+  }
+   @Test
+  def testVarAssign():Unit={
+    runTest("""
+          |var a:Int=4
+          |def f():Unit = {
+          |  println(a)
+          |  a/**/=5
+          |}
+          |""".stripMargin,
+          "var a: Int")
+  }
+
+  @Test
+  def testVarDeclr():Unit={
+    runTest("""
+          |var a/**/:Int=4
+          |def f():Unit = {
+          |  println(a)
+          |}
+          |""".stripMargin,
+          "var a: Int")
+  }
+
+  private def runTest(in: String, expected: String): Unit = {
```

Comment:
Could you please rename this method to something like `runTestWithMarker`? This makes it easier to understand the difference between the two `runTest` methods.